brainkeeper/src/app/app.component.spec.ts

```diff
@@ -6,10 +6,10 @@ describe('AppComponent', () => {
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       imports: [
-        RouterTestingModule
+        RouterTestingModule,
```

Comment:
Do arrays have comma at the last item in our style guides ?

Reply:
Please look at the conversations above

Reply:
thank, I overlooked them